src/main/java/appeng/core/AEConfig.java

```diff
@@ -506,4 +550,217 @@ public void nextPowerUnit( final boolean backwards )
+	{
+		return disableColoredCableRecipesInJEI;
+	}
+
+	public int getCraftingCalculationTimePerTick()
+	{
+		return craftingCalculationTimePerTick;
+	}
+
+	public PowerUnits getSelectedPowerUnit()
+	{
+		return selectedPowerUnit;
+	}
+
+	public int[] getCraftByStacks()
+	{
+		return craftByStacks;
+	}
+
+	public int[] getPriorityByStacks()
+	{
+		return priorityByStacks;
+	}
+
+	public int[] getLevelByStacks()
+	{
+		return levelByStacks;
+	}
+
+	public int getStorageBiomeID()
+	{
+		return storageBiomeID;
+	}
+
+	public int getStorageProviderID()
+	{
+		return storageProviderID;
+	}
+
+	public double getSpatialPowerExponent()
+	{
+		return spatialPowerExponent;
+	}
+
+	public double getSpatialPowerMultiplier()
+	{
+		return spatialPowerMultiplier;
+	}
+
+	public String[] getGrinderOres()
+	{
+		return grinderOres;
+	}
+
+	public double getOreDoublePercentage()
+	{
+		return oreDoublePercentage;
+	}
+
+	public int getWirelessTerminalBattery()
+	{
+		return wirelessTerminalBattery;
+	}
+
+	public int getEntropyManipulatorBattery()
+	{
+		return entropyManipulatorBattery;
+	}
+
+	public int getMatterCannonBattery()
+	{
+		return matterCannonBattery;
+	}
+
+	public int getPortableCellBattery()
+	{
+		return portableCellBattery;
+	}
+
+	public int getColorApplicatorBattery()
+	{
+		return colorApplicatorBattery;
+	}
+
+	public int getChargedStaffBattery()
+	{
+		return chargedStaffBattery;
+	}
+
+	public float getSpawnChargedChance()
+	{
+		return spawnChargedChance;
+	}
+
+	public int getQuartzOresPerCluster()
+	{
+		return quartzOresPerCluster;
+	}
+
+	public int getQuartzOresClusterAmount()
+	{
+		return quartzOresClusterAmount;
+	}
+
+	public int getChargedChange()
+	{
+		return chargedChange;
+	}
+
+	public int getMinMeteoriteDistance()
+	{
+		return minMeteoriteDistance;
+	}
+
+	public int getMinMeteoriteDistanceSq()
+	{
+		return minMeteoriteDistanceSq;
+	}
+
+	public double getMeteoriteClusterChance()
+	{
+		return meteoriteClusterChance;
+	}
+
+	public int getMeteoriteMaximumSpawnHeight()
+	{
+		return meteoriteMaximumSpawnHeight;
+	}
+
+	public int[] getMeteoriteDimensionWhitelist()
+	{
+		return meteoriteDimensionWhitelist;
+	}
+
+	public double getWirelessBaseCost()
+	{
+		return wirelessBaseCost;
+	}
+
+	public double getWirelessCostMultiplier()
+	{
+		return wirelessCostMultiplier;
+	}
+
+	public double getWirelessTerminalDrainMultiplier()
+	{
+		return wirelessTerminalDrainMultiplier;
+	}
+
+	public double getWirelessBaseRange()
+	{
+		return wirelessBaseRange;
+	}
+
+	public double getWirelessBoosterRangeMultiplier()
+	{
+		return wirelessBoosterRangeMultiplier;
+	}
+
+	public double getWirelessBoosterExp()
+	{
+		return wirelessBoosterExp;
+	}
+
+	public double getWirelessHighWirelessCount()
+	{
+		return wirelessHighWirelessCount;
+	}
+
+	// Setters keep visibility as low as possible.
+
+	void setStorageBiomeID( int id )
+	{
+		this.storageBiomeID = id;
+	}
+
+	void setStorageProviderID(int id)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '(' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)